cucumber.js

```diff
@@ -76,6 +76,7 @@ module.exports = {
     worldParameters: {
       apiUrl: 'http://localhost:8080/api/v1/opvragenBevoegdheidTotGezag',
       api: 'gezag-api',
+      tagScenariosAsDeprecated: true,
```

Comment:
is er een reden om nu alleen bij gezag deze parameter toe te voegen en nog niet bij o.a. DataApiDeprecated?

Reply:
geen specifieke reden om wel/niet toe te voegen. Het is nu toegevoegd aan Info- en DataApiDeprecated scenarios